addons/im_livechat/security/ir.model.access.csv

```diff
@@ -7,3 +7,14 @@ access_livechat_support_report_operator,im_livechat.report.operator,model_im_liv
 access_livechat_channel_rule,im_livechat.channel.rule,model_im_livechat_channel_rule,,1,0,0,0
 access_livechat_channel_rule_user,im_livechat.channel.rule,model_im_livechat_channel_rule,im_livechat_group_user,1,1,1,0
 access_livechat_channel_rule_manager,im_livechat.channel.rule,model_im_livechat_channel_rule,im_livechat_group_manager,1,1,1,1
+access_livechat_chatbot,im_livechat.chatbot.script,model_im_livechat_chatbot_script,,1,0,0,0
+access_livechat_chatbot_user,im_livechat.chatbot.script.user,model_im_livechat_chatbot_script,im_livechat_group_user,1,1,1,0
+access_livechat_chatbot_manager,im_livechat.chatbot.script.manager,model_im_livechat_chatbot_script,im_livechat_group_manager,1,1,1,1
+access_livechat_chatbot_script_question,im_livechat.chatbot.script_question,model_im_livechat_chatbot_script_step,,1,0,0,0
+access_livechat_chatbot_script_question_user,im_livechat.chatbot.script_question.user,model_im_livechat_chatbot_script_step,im_livechat_group_user,1,1,1,0
+access_livechat_chatbot_script_question_manager,im_livechat.chatbot.script_question.manager,model_im_livechat_chatbot_script_step,im_livechat_group_manager,1,1,1,1
+access_livechat_chatbot_script_question_answer,im_livechat.chatbot.script_question_answer,model_im_livechat_chatbot_script_question_answer,,1,0,0,0
+access_livechat_chatbot_script_question_answer_user,im_livechat.chatbot.script_question_answer.user,model_im_livechat_chatbot_script_question_answer,im_livechat_group_user,1,1,1,0
+access_livechat_chatbot_script_question_answer_manager,im_livechat.chatbot.script_question_answer.manager,model_im_livechat_chatbot_script_question_answer,im_livechat_group_manager,1,1,1,1
+access_livechat_chatbot_mail_message_all,im_livechat.chatbot.mail.message,model_im_livechat_chatbot_mail_message,,1,0,0,0
+access_livechat_chatbot_mail_message_user,im_livechat.chatbot.mail.message,model_im_livechat_chatbot_mail_message,base.group_portal,1,1,1,1
```

Comment:
Seems like portal has an open bar ^^ . Would remove access to it, as anyway code will be executed through controllers. What do you think ?